rocAL/rocAL/source/meta_node_rotate.cpp

```diff
@@ -89,20 +90,20 @@ void RotateMetaNode::update_parameters(MetaDataBatch* input_meta_data)
             min_y = std::min(y1, std::min(y2, std::min(y3, y4)));
             max_x = std::max(x1, std::max(x2, std::max(x3, x4)));
             max_y = std::max(y1, std::max(y2, std::max(y3, y4)));
-            box.x = (min_x > 0) ? min_x : 0;
-            box.y = (min_y > 0) ? min_y : 0;
-            box.w = max_x - box.x;
-            box.h = max_y - box.y;
+            box.l = (min_x > 0) ? min_x : 0;
+            box.t = (min_y > 0) ? min_y : 0;
+            box.r = max_x ;
+            box.b = max_y ;
             if (BBoxIntersectionOverUnion(box, dest_image) >= _iou_threshold)
             {
-                float xA = std::max(dest_image.x, box.x);
-                float yA = std::max(dest_image.y, box.y);
-                float xB = std::min(dest_image.x + dest_image.w, box.x + box.w);
-                float yB = std::min(dest_image.y + dest_image.h, box.y + box.h);
-                box.x = xA;
-                box.y = yA;
-                box.w = xB - xA;
-                box.h = yB - yA;
+                float xA = std::max(dest_image.l, box.l);
+                float yA = std::max(dest_image.t, box.t);
+                float xB = std::min(dest_image.r, box.r);
+                float yB = std::min(dest_image.b, box.b);
+                box.l = xA;
```

Comment:
do inline compute and remove extra variables

Reply:
Resolved